website/views/shared/profiles/achievements.jade

```diff
@@ -146,8 +146,8 @@ div(ng-if=':: profile.achievements.originalUser')
   
 div(ng-if='profile.achievements.challenges || user._id == profile._id')
   // This is a very strange icon to use. revisit
-  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges')
-  div(ng-class='{muted: !profile.achievements.challenges}')
+  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges.length')
```

Comment:
Also, it should be safe to add the one time binding marking (`::`) to both of them, so:
``` jade
.achievement.achievement-karaoke(ng-if='::profile.achievements.challenges.length')
div(ng-class='::{muted: !profile.achievements.challenges.length}')
```